### PR TITLE
Шатров Виктор гр. 2538

### PR DESCRIPTION
**Pull request by vi34**
Photos from Flickr daily most interesting photostream.
extra features also completed(save, set as wallpaper, open in browser)
Also pictures in fullscreen mode are scalable.

### CODE REVIEW

File: FlickrV/app/src/main/java/com/example/vi34/flickrv/FullImageActivity.java [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    }
+
+    private Boolean checkNet() {
+        ConnectivityManager connMgr = (ConnectivityManager)
+                getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo networkInfo = connMgr.getActiveNetworkInfo();
+        if (networkInfo != null && networkInfo.isConnected()) {
+            return true;
+        } else {
+            Toast.makeText(this, "Check your Internet Connection", Toast.LENGTH_SHORT).show();
```
Comment (vi34) [inline FlickrV/app/src/main/java/com/example/vi34/flickrv/FullImageActivity.java:145]: Sorry, I forgot to move strings from Toasts to resources

File: FlickrV/app/src/main/java/com/example/vi34/flickrv/MyPhoto.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import java.io.ByteArrayInputStream;
+
+/**
+ * Created by vi34 on 17.01.15.
+ */
+public class MyPhoto {
+    String author;
+    String fullUrl;
+    String browseUrl;
```
Comment (svasilinets) [inline FlickrV/app/src/main/java/com/example/vi34/flickrv/MyPhoto.java:14]: modifiers?

File: FlickrV/app/src/main/java/com/example/vi34/flickrv/MyIntentService.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                        try {
+                            wallpaperManager.setBitmap(bmp);
+                        } catch (IOException e) {
+                            e.printStackTrace();
+                        }
+                    } else {
+
+                        try {
+                            String root = Environment.getExternalStorageDirectory().toString();
+                            File myDir = new File(root + "/saved_images");
```
Comment (svasilinets) [inline FlickrV/app/src/main/java/com/example/vi34/flickrv/MyIntentService.java:160]: you can use other constructor here, new File(dir, fileName)

**Comment by svasilinets** [comment]
- 1. Inaccurate Code
- 1. Inaccurate design.
- 3. Landscape mode works bad: empty screen after orientation change
  20 - 1 - 1 - 3 = 15.

**Comment by svasilinets** [comment]
Ok. +2 for scaleable image

17

**Pull request closed without merging**